daemon/daemon.go

```diff
@@ -1056,7 +1058,10 @@ func writeDistributionProgress(cancelFunc func(), outStream io.Writer, progressC
 
 	for prog := range progressChan {
 		if err := progressOutput.WriteProgress(prog); err != nil && !operationCancelled {
-			logrus.Errorf("error writing progress to client: %v", err)
+			// don't log broken pipe errors as this is the normal case when a client aborts
+			if !isBrokenPipe(err) {
+				logrus.Errorf("error writing progress to client: %v", err)
```

Comment:
While I agree its not an "error" I do wonder if its worth logging at some level so that someone looking at the daemon logs knows that the download was never completed and that its because the client went away. Perhaps an info message?